docs/pages/desktop-access/directory-sharing.mdx

```diff
@@ -0,0 +1,188 @@
+the directory from the Teleport Web UI.
+
+## Step 3/4. Disconnect
+
+To stop sharing a directory, end your remote desktop session. Navigate to the
+three-dot menu on the upper-right of the desktop session in your browser and
+click "Disconnect." 
+
+The next time you start a session on the remote desktop, the directory will no
+longer be shared. You will need to share the directory again to access its
+content.
+
+## Step 4/4. Disable Directory Sharing
+
+The Teleport Auth Service grants access to Directory Sharing on a remote desktop
+based on the roles of the user who initiates the desktop session. 
+
+A Teleport role enables Directory Sharing by default. If one of a user's
+Teleport roles disables Directory Sharing, then Directory Sharing will be
+disabled for that user.
+
+To disable Directory Sharing for a Teleport user, define a role similar to the
+following in a file called `role.yaml`:
+
+```yaml
+kind: role
+version: v5
+metadata:
+  name: "no-sharing"
+spec:
+  options:
+    desktop_directory_sharing: false
+```
+
+Create the role:
+
+```code
+$ tctl create -f role.yaml
+```
+
+Retrieve the role for the Teleport user you want to modify and save their role
+definition in `user.yaml`. Here we assume the user is called `myser`:
+
+```code
+$ tctl get user/myuser > user.yaml
+```
+
+Edit `user.yaml` to add the new role:
+
+```diff
+spec:
+  - access
+  - editor
+  - auditor
++ - no-sharing
+  status:
+    is_locked:false
+```
+
+Apply your changes:
+
+```code
+$ tctl create -f role.yaml
+$ tctl create -f user.yaml
+```
+
+## Next steps
+
+### Further reading
+
+Directory Sharing is a powerful tool for editing files on remote desktops, and
+you'll want to make sure you have a comprehensive audit trail so you can conduct
+a post-incident retrospective or investigate unintended usage. Learn how to set
+up [session recording for Desktop Access](./reference/sessions.mdx).
+
+Aside from Directory Sharing, Desktop Access also enables you to share the
+contents of your clipboard with a remote desktop. Learn how to use [Clipboard
+Sharing](./reference/clipboard.mdx).
+
+### How Directory Sharing works
+
+Directory Sharing involves a browser on a local workstation and a remote Windows
+desktop.
+
+On the remote side, Directory Sharing takes advantage of [file system-related
+messaging](https://docs.microsoft.com/en-us/openspecs/windows_protocols/ms-rdpefs/)
+within the Remote Desktop Protocol (RDP). On the local side, the Teleport Web UI
+uses the browser's [File System Access
+API](https://developer.mozilla.org/en-US/docs/Web/API/File_System_Access_API) to
+read from and write to a user-selected local directory.
+
+The Teleport Web UI establishes a secure WebSocket session with the Teleport
+Proxy Service, which forwards traffic to and from the relevant Teleport Desktop
+Service instance. 
+
+The Teleport Desktop Service converts traffic from the Teleport Web UI into RDP
```

Comment:
```suggestion
The Teleport Desktop Service converts [TDP](https://github.com/gravitational/teleport/blob/master/rfd/0067-desktop-access-file-system-sharing.md) traffic from the Teleport Web UI into RDP
```

Reply:
Added. I put a brief explanation of TDP before this paragraph and moved the link to a "You can read more..." paragraph at the end of this section to make the link target more explicit